src/App.js

```diff
@@ -37,9 +37,31 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+
+      <Tweet tweet={ tweetsArray[0] }
+      image = {tweetsArray[0].user.image}
+      handle={tweetsArray[0].user.handle}
+      timestamp={tweetsArray[0].timestamp}
+      message={tweetsArray[0].message} 
+      />
```

Comment:
Always try to leave App.js quite clean. Pas a object into a component , and conttinue object destructuring as you have done already in Tweet.js